.gitlab-ci.yml

```diff
@@ -7,9 +7,13 @@ include:
   file: private.yml
 
 cache:
-  key: ${CI_COMMIT_REF_SLUG}
+  key:
+    files:
+      - package.json
+    prefix: ${CI_COMMIT_REF_SLUG}
   paths:
     - .npm/
```

Comment:
I'm yet to encounter a `.npm` directory. What is it used for?

Reply:
https://docs.gitlab.com/ee/ci/caching/#cache-nodejs-dependencies